stuffer/s2n_stuffer_text.c

```diff
@@ -43,24 +43,28 @@ int s2n_stuffer_peek_check_for_str(struct s2n_stuffer *s2n_stuffer, const char *
     return 0;
 }
 
-int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer)
+int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer, uint32_t *skipped)
 {
-    int skipped = 0;
+    PRECONDITION_POSIX(s2n_stuffer_is_valid(s2n_stuffer));
+    uint32_t skip = 0;
     while (s2n_stuffer->read_cursor < s2n_stuffer->write_cursor) {
         switch (s2n_stuffer->blob.data[s2n_stuffer->read_cursor]) {
         case ' ':              /* We don't use isspace, because it changes under locales */
         case '\t':
         case '\n':
         case '\r':
             s2n_stuffer->read_cursor += 1;
-            skipped += 1;
+            skip += 1;
             break;
         default:
-            return skipped;
+            if(skipped != NULL) *skipped = skip;
```

Comment:
what about a `goto finished;` to remove the duplicate code? Or adding a boolean to the while loop so you could break out once you run out of whitespace.

Reply:
Done.